### PR TITLE
warm up for test_kafka_json_as_string_no_kdc

### PR DESCRIPTION
**Pull request by ilejn**
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Detailed description / Documentation draft:
Addresses https://github.com/ClickHouse/ClickHouse/issues/32644

A bug  in test_kafka_json_as_string_no_kdc fixed.
When the test is run alone (not preceded by any other kerberized kafka test), the assertion instance.contains_in_log("Ticket expired") is failed because there is no ticket at all.

### CODE REVIEW

**Review by alexey-milovidov — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
@mergifyio update

**Comment by ilejn** [comment]
@alexey-milovidov  It seems https://github.community/t/how-can-we-enable-allow-edits-from-maintainers-by-default/2847 that if a fork is a non-personal (but org) account there is no way to Allow edits from maintainers.
No idea what to do yet.

**Comment by alesapin** [comment]
Please, push something to this branch or merge with master manually.

File: tests/integration/test_storage_kerberized_kafka/test.py [hunk 1]
```diff
@@ -97,6 +97,19 @@ def test_kafka_json_as_string(kafka_cluster):
     assert instance.contains_in_log("Parsing of message (topic: kafka_json_as_string, partition: 0, offset: 1) return no rows")
 
 def test_kafka_json_as_string_no_kdc(kafka_cluster):
+    instance.query('''
```
Comment (azat) [inline tests/integration/test_storage_kerberized_kafka/test.py:100]: Worth adding a comment why this unused table is required (to create message in log).

Comment (ilejn) [reply tests/integration/test_storage_kerberized_kafka/test.py:103]: Ok.

**Pull request merged**